injective/assetlist.json

```diff
@@ -174,6 +174,34 @@
         }
       ]
     },
+    {
+      "description": "The most degenerate NFT on Injective. Gravedigger collection for $bINJ. Giving power back to the community.",
```

Comment:
Want to double check that this is the right description?

Reply:
the description is correct, thank you